packages/core/src/services/users/createUser.ts

```diff
@@ -29,4 +30,15 @@ export async function createUser(
 
     return Result.ok(user)
   }, db)
+
+  if (result.ok) {
+    publisher.publishLater({
+      type: 'claimReferralInvitations',
```

Comment:
So this event is always triggered? In all users creations?

Reply:
Yes! When a user is created, it checks if there's any referral claimed for that email, and accepts them. I moved this logic to a job instead of the same `createUser` service